js/module-details/index.js

```diff
@@ -13,8 +14,10 @@ function getVersion (baseDir) {
   }
 }
 
+const getDetailsWithSep = path.sep === '/' ? getDetails : (f) => getDetails(f.replaceAll(path.sep, '/'))
```

Comment:
This will fail, should be other way around:
```
const getDetailsWithSep = path.sep === '/' ? getDetails : (f) => getDetails(f.replaceAll('/', path.sep))
``
getDetails method expects `path.sep` and in windows we are sending `/`.